cognitedata/reveal
# fix: Issue with incorrect wheel events coordinated in StationaryCameraManager (360 images camera manager)

#### Type of change

![Bug](https://img.shields.io/badge/Type-Bug-red) 


#### Jira ticket :blue_book:


https://cognitedata.atlassian.net/browse/REV-727

## Description :pencil:

`StationaryCameraManager` was using incorrect mouse coordinates in wheel event handler which resulted in zooming to incorrect location when Reveal window was not taking the whole space of the page. This PR uses the same approach as in wheel events handler from `DefaultCameraManager` that takes into account offset of DOM element with reveal.

## How has this been tested? :mag:


In Viewer example with loaded 360 images and locally on android phone with initialisation of `StationaryCameraManager` in the example.

## Test instructions :information_source:
1. Open viewer example with any model on `3d-test` project.
2. In 360 Images menu section load image set with site id `helideck-site-2`. Enter any of loaded 360 images.
3. In Viewer size section of the menu, set the value to "quarter size"
4. Zoom into any part of 360 image and notice that it is zooming to the place exactly under the cursor which was not the case before this PR.



## Checklist :ballot_box_with_check:
